content/docs/rfcs/21/README.md

```diff
@@ -2,7 +2,7 @@
 slug: 21
 title: 21/WAKU2-FAULT-TOLERANT-STORE
 name: Waku v2 Fault-Tolerant Store
-status: raw
+status: draft
```

Comment:
Missing place change in menu as well

Is everything else up to date in this spec or any major mismatch with implementation?

Reply:
Thanks @oskarth , done in https://github.com/vacp2p/rfc/pull/459/commits/72ca86577be755e1a440af0ef1b87a51f2078604

I was also scrolling over the past issues and realized I could address one of them https://github.com/status-im/nim-waku/issues/706 in this PR. Please see this [commit](https://github.com/vacp2p/rfc/pull/459/commits/6ba429f80eb76c730a205f830a6c1ca1ea881a00)

Reply:
Everything else re FT-store spec is up to date